riscv_config/schemaValidator.py

```diff
@@ -391,3 +391,7 @@ def _check_with_sedeleg_reset(self, field, value):
         s = format(value, '#{}b'.format(supported_xlen[0] + 2))
         if (s[-11:-8]) != '000' and value >= int("400", 16):
             self._error(field, " 11,10,9 bits should be hardwired to 0")
+
+    def _check_with_p_check(self, field, value):
+        if not 'Zpn' in self.document['ISA']: #Zpn is mandatory for P-SIMD
```

Comment:
The condition should also account for the `implemented` field in the node. Similar to [this](https://github.com/riscv-software-src/riscv-config/blob/af909b1f3e32286bcb552af71b819f692f1982ac/riscv_config/schemaValidator.py#L307).

Reply:
Not sure about the "error statement to display" when 'implemented' is false. 
```
        check = False
        if 'implemented' in value:
            if value['implemented']:
                check = True

        elif not check:
            self._error(field, "Check whether 'implemented' field is set to true")
```